payments/mobile-pay/redirect.md

```diff
@@ -70,35 +71,187 @@ complete purchase.
 
 ```mermaid
 sequenceDiagram
-  Consumer->>Merchant: start purchase
-  Activate Merchant
-  Merchant->>PayEx: POST <mobilepay payment> (operation=PURCHASE)
+  Consumer->>Merchant: start purchase (pay with MobilePay)
+  activate Merchant
+
+  Merchant->>SwedbankPay: POST <Create MobilePay payment>
   note left of Merchant: First API request
-  Activate PayEx
-  PayEx-->>Merchant: payment resource
-  Deactivate PayEx
-  Merchant-->>Consumer: redirect to authorize page
-  Deactivate Merchant
-  note left of Consumer: redirect to PayEx
-
-  Consumer->>PayEx: enter mobilepay info
-  Activate PayEx
-  PayEx->>Consumer_App: Confirm payment
-  Consumer_App-->>PayEx: Payment confirmed
-  Deactivate PayEx
-
-  PayEx-->>Consumer: redirect to merchant
-  note left of PayEx: redirect back to merchant
-
-  Consumer->>Merchant: access merchant page
-  Activate Merchant
-  Merchant->>PayEx: GET <mobilepay payment>
+  activate SwedbankPay
+  SwedbankPay-->>Merchant: payment resource
+  deactivate SwedbankPay
+  SwedbankPay -->> SwedbankPay: Create payment
+  Merchant-->>Consumer: Redirect to payment page
+  note left of Consumer: redirect to MobilePay
+  Consumer-->>SwedbankPay: enter mobile number
+  activate SwedbankPay
+
+  SwedbankPay-->>MobilePay_API: Initialize MobilePay payment
```

Comment:
Please provide an alias for `MobilePay_API` with the `as` keyword in the beginning of the sequence diagram.